### PR TITLE
UI deploy tool, UI routes, UI files

### PR DESCRIPTION
**Pull request by aradnaev**
added: UI deploy tool, UI routes, UI files.
to test: python deploy_ui.py
(assumes ui repo is in ~/projects/etabot/etabot-ui/)

### CODE REVIEW

File: etabotsite/etabotapp/misc_utils/django_indexhtml_mod.py [hunk 1]
```diff
@@ -16,10 +16,12 @@
 logger = logging.getLogger()
 logger.setLevel(logging.DEBUG)
 
+
 def html_to_django_template(
-    src_html,
-    target_html,
-    django_static_subpath):
+        src_html,
```
Comment (ShanshanHe) [inline etabotsite/etabotapp/misc_utils/django_indexhtml_mod.py]: please revert the indent changes here.

File: etabotsite/etabotsite/settings.py [hunk 2]
```diff
@@ -153,4 +152,4 @@
 #SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 #SECURE_SSL_REDIRECT = True
 #SESSION_COOKIE_SECURE = True
-#CSRF_COOKIE_SECURE = True
+CSRF_COOKIE_SECURE = False
```
Comment (ShanshanHe) [inline etabotsite/etabotsite/settings.py]: This is not secure to make it false in production.

File: etabotsite/etabotsite/settings.py [hunk 3]
```diff
@@ -74,7 +73,7 @@
     
 ]
 
-CORS_ORIGIN_ALLOW_ALL = True   
+# CORS_ORIGIN_ALLOW_ALL = True
```
Comment (ShanshanHe) [inline etabotsite/etabotsite/settings.py]: Why is it commented away?

File: etabotsite/templates/index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{% load staticfiles %}
```
Comment (ShanshanHe) [inline etabotsite/templates/index.html]: Are you sure you want to add those `index.html` and the static files into pmp repo?

File: etabotsite/etabotapp/views.py [hunk 5]
```diff
@@ -6,6 +6,17 @@
 from .models import Project
 from .models import TMS
 from .permissions import IsOwner
+from django.views.decorators.csrf import ensure_csrf_cookie
```
Comment (ShanshanHe) [inline etabotsite/etabotapp/views.py]: Is the import used anywhere in the file? If not, please remove.

File: etabotsite/etabotapp/deploy_ui.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import misc_utils.ng2plus_dist2django as ng2django
+
+ng2django.deploy_ng2plus(
+    '~/projects/etabot/etabot-ui/dist/',
```
Comment (ShanshanHe) [inline etabotsite/etabotapp/deploy_ui.py]: This is an absolute path which should not appear in production code. Not everybody has `~/project/etabot/etabot-ui/dist` directory on their laptop.

File: etabotsite/etabotsite/settings.py (hunk 2 above)
Comment (aradnaev) [reply etabotsite/etabotsite/settings.py]: this is to allow csrf cookies to be sent over http - this is temporary while we are setting up HTTPS

File: etabotsite/etabotsite/settings.py (hunk 3 above)
Comment (aradnaev) [reply etabotsite/etabotsite/settings.py]: this is for security to forbid cross-origin requests

File: etabotsite/etabotapp/views.py (hunk 5 above)
Comment (aradnaev) [reply etabotsite/etabotapp/views.py]: yes, using it for index view

File: etabotsite/templates/index.html (hunk 4 above)
Comment (aradnaev) [reply etabotsite/templates/index.html]: yes, I think it is ok as an example, and it helps with our testing.

**Pull request merged**